twilio/http/http_client.py

```diff
@@ -47,4 +50,7 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
             timeout=timeout,
         )
 
+        self.last_request = request
```

Comment:
can we set this before we send the request? That way if `session.send` raises an exception it will still be set.